db/migrate/20170725120715_add_gender_to_user.rb

```diff
@@ -0,0 +1,17 @@
+class AddGenderToUser < ActiveRecord::Migration[5.0]
+  def up
+    execute <<-SQL
```

Comment:
http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html

Here you can find migration example. Also typing 'rails enum' give a variety of nice articles.